sycl/include/sycl/ext/intel/experimental/esimd/detail/math_intrin.hpp

```diff
@@ -457,14 +434,15 @@ __esimd_dpas_bits_precision(__ESIMD_ENS::argument_type precisionType) {
              : 1;
 }
 
-template <__ESIMD_ENS::argument_type src1_precision,
-          __ESIMD_ENS::argument_type src2_precision, int systolic_depth,
-          int repeat_count, typename RT, typename T1, typename T2,
-          __ESIMD_NS::uint SZ, __ESIMD_NS::uint N1, __ESIMD_NS::uint N2>
+template <typename RT, typename T1, typename T2, __ESIMD_NS::uint SZ,
+          __ESIMD_NS::uint N1, __ESIMD_NS::uint N2>
 inline __ESIMD_DNS::vector_type_t<RT, SZ>
 __esimd_dpas_inner(const __ESIMD_DNS::vector_type_t<RT, SZ> *src0,
                    const __ESIMD_DNS::vector_type_t<T1, N1> &src1,
-                   const __ESIMD_DNS::vector_type_t<T2, N2> &src2) {
+                   const __ESIMD_DNS::vector_type_t<T2, N2> &src2,
+                   __ESIMD_ENS::argument_type src1_precision,
```

Comment:
this should probably return to compile-time evaluation

Reply:
Change is removed.